ts/parts-map/HeatmapSeries.ts

```diff
@@ -261,6 +283,30 @@ seriesType<Highcharts.HeatmapSeriesOptions>(
                  * rule.
                  */
                 brightness: 0.2
+            },
+            inactive: {
+                /**
+                 * When hovering over a cell, decide which **other** cells
+                 * should not apply an inactive state.
+                 * For example `'column-row'` will apply inactive state to all
+                 * other cells except cells within the same row or column.
+                 *
+                 * Possible options:
+                 * - `'all'`- all cells (aka disabled)
+                 * - `'column'` - all cells within the same row
+                 * - `'row'` - all cells within the same column
+                 * - `'column-row'` - all cells within the same row or column
+                 * - `'self'` - only current cell
+                 * Point padding for a single point.
+                 *
+                 * @sample         highcharts/series-heatmap/inactive-except/
+                 *                 Compare different `except` options
+                 *
+                 * @validvalue     ["all", "column", "row", "column-row", "self"]
+                 * @since          7.1.4
+                 * @product        highcharts highmaps
+                 */
+                except: 'column-row'
```

Comment:
> Why not simply filter?
> in tree-structure, we don't loop over all points

Not completely convinced about the naming. Would it be wrong to call it `filter` even though it is not looping all points?

Also some alternative synonyms to `except` are `omit`, `exclude`, `skip`. Not sure if they are any better though.

Reply:
Personally, I like `exclude` name. I'm also worried that users may expect `filter` to be a function. Like `array.filter(fun)`.